@here/harp-mapview/lib/Tile.ts

```diff
@@ -949,992 +828,17 @@ export class Tile implements CachedResource {
-
-                    this.registerTileObject(outlineObj);
-                    objects.push(outlineObj);
-                }
-
-                // Add the fill area edges as a separate geometry.
-                if (hasSolidLinesOutlines) {
-                    const outlineTechnique = technique as SolidLineTechnique;
-                    const outlineMaterial = material.clone() as SolidLineMaterial;
-                    const outlineColor = ColorCache.instance.getColor(
-                        getOptionValue(
-                            getPropertyValue(
-                                outlineTechnique.secondaryColor,
-                                Math.floor(this.mapView.zoomLevel)
-                            ),
-                            "#000000"
-                        )
-                    );
-                    outlineMaterial.uniforms.diffuse.value = outlineColor;
-                    const outlineObj = new ObjectCtor(bufferGeometry, outlineMaterial);
-
-                    outlineObj.renderOrder =
-                        outlineTechnique.secondaryRenderOrder !== undefined
-                            ? outlineTechnique.secondaryRenderOrder
-                            : technique.renderOrder - 0.0000001;
-
-                    if (group.renderOrderOffset !== undefined) {
-                        outlineObj.renderOrder += group.renderOrderOffset;
-                    }
-
-                    const fadingParams = this.getFadingParams(technique);
-                    FadingFeature.addRenderHelper(
-                        outlineObj,
-                        fadingParams.fadeNear,
-                        fadingParams.fadeFar,
-                        true,
-                        false,
-                        (renderer, mat) => {
-                            const lineMaterial = mat as SolidLineMaterial;
-
-                            const metricUnits = getPropertyValue(
-                                outlineTechnique.metricUnit,
-                                this.tileKey.level
-                            );
-                            const unitFactor =
-                                metricUnits === "Pixel" ? this.mapView.pixelToWorld * 0.5 : 1.0;
-
-                            lineMaterial.lineWidth =
-                                getOptionValue(
-                                    getPropertyValue(
-                                        outlineTechnique.secondaryWidth,
-                                        this.mapView.zoomLevel
-                                    ),
-                                    SolidLineMaterial.DEFAULT_WIDTH
-                                ) * unitFactor;
-                        }
-                    );
-
-                    this.registerTileObject(outlineObj);
-                    objects.push(outlineObj);
-                }
-            }
-        }
-    }
-
-    /**
-     * Counts the number of vertices in a tile.
-     */
-    countVertices(): number {
```

Comment:
I notice that this didn't survive the transition to GeometryLoader?

Reply:
It was not used before, the resource measurement works differently now.